libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -273,7 +274,20 @@ class MapboxNavigation(
 
     private var reachabilityObserverId: Long? = null
 
+    private var isDestroyed = false
```

Comment:
Do we really need `isDestroyed`? Apparently `hasInstance` has the same lifecycle 🤔